charts/newrelic-infrastructure/templates/_helpers.tpl

```diff
@@ -152,6 +152,24 @@ Returns fargate
 {{- end -}}
 {{- end -}}
 
+{{/*
+Returns the updateStrategy, either .Values.updateStrategy directly if it is an object, or wrapped if it is a string
+This is done to keep compatibility with old values and --reuse-values.
+*/}}
+{{- define "newrelic.updateStrategy" -}}
+{{- if .Values.updateStrategy }}
+{{- if eq "string" (printf "%T" .Values.updateStrategy) }}
+updateStrategy:
+  type: {{ .Values.updateStrategy }}
+  rollingUpdate:
```

Comment:
isn't needed to check if the update strategy was rollingUpdate?

Reply:
Theoretically yes, but this is as it was hardcoded before 🤔 
I can do the check here though, just to be in the safe side.

Reply:
oh i didn't know that, otherwise I would just complete the `type` and nothing else thinking that this would be using the default values for rollingUpdate and recreate

Reply:
if that was hardcoded befor , i don't think that needs the check